bundle-core/src/main/java/net/discdd/utils/FileUtils.java

```diff
@@ -34,8 +33,8 @@ public static void createEmptyFileIfNeeded(Path path) throws IOException {
 
     public static OutputStream getFilePath(BundleDownloadResponse response, Path receive_Directory) throws IOException {
         String fileName = response.getMetadata().getBid();
-        var directoryReceive = receive_Directory.resolve(response.getMetadata().getSenderId());
-
+        // TODO: this only works for the TRANSPORT type
```

Comment:
this will actually work for client type now, since for download from server, bundles are now stored in subdirectories based on clientId.